pilot/pkg/model/context.go

```diff
@@ -327,8 +327,8 @@ type Proxy struct {
 	// XdsNode is the xDS node identifier
 	XdsNode *core.Node
 
-	WorkloadEntryName        string
-	WorkloadEntryAutoCreated bool
+	WorkloadEntryName           string
+	WorkloadEntryAutoRegistered bool
```

Comment:
`AutoCreated` is renamed into `AutoRegistered` based on [feedback](https://github.com/istio/istio/pull/44712#pullrequestreview-1414150727) in https://github.com/istio/istio/pull/44712